src/components/ResponsivePage/ResponsiveContainer.stories.tsx

```diff
@@ -0,0 +1,87 @@
+import type { Meta, StoryObj } from '@storybook/react';
```

Comment:
What's the point of storybook for us again? I have heard of it, but didn't dig too deep.

Reply:
Storybook is used to keep track of the components outside of the development environment. Its easy to look through and see what has been made for the frontend.